<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Room_add</class>
 <widget class="QDialog" name="Room_add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Номер комнаты</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="le_number"/>
    </item>
   </layout>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>521</width>
     <height>311</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>519</width>
      <height>309</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>440</y>
     <width>164</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Характеристики</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Room_add</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Room_add</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
